src/handlers/localstorage/globalSettings.ts

```diff
@@ -64,3 +65,16 @@ export const saveTransactionSignatures = (transactionSignatures: any) =>
  */
 export const getTransactionSignatures = () =>
   getGlobal(TRANSACTION_SIGNATURES, {});
+
+/**
+ * @desc get testnets enabled preference
+ */
+export const getTestnetsEnabled = () => getGlobal(TESTNETS_ENABLED, false);
```

Comment:
if this is defaulted to false, what happens to the users who had switched to a testnet before upgrading

Reply:
they will still be in testnets but can still go to settings -> dev settings to fix. imo this is fine, testnets are so broken rn so the user % is prob really small, can be handled by support if they can't find